packages/portal/spec-details/sandbox/App.vue

```diff
@@ -10,21 +10,39 @@
           }"
         >
           <h3>Try it Out:</h3>
-          <label>
-            <input
-              v-model="hasSidebar"
-              type="checkbox"
-            > hasSidebar
-          </label>
-          <label class="ml-3">
-            <input
-              v-model="essentialsOnly"
-              type="checkbox"
-            > essentialsOnly
-          </label>
+          <div class="props">
+            <label>
+              <input
+                v-model="hasSidebar"
+                type="checkbox"
+              > hasSidebar
+            </label>
+            <label class="ml-3">
```

Comment:
issue: you cannot utilize Kongponents utility classes inside this repository. Please replace this and all Kongponents classes with native CSS in the `style` block.

Reply:
this was a copy-pasted fragment, but I'll adjust it shortly!

Reply:
Sorry! That's my code, apologies

Reply:
fixed